.eslintrc

```diff
@@ -0,0 +1,20 @@
+---
+extends: react-app
```

Comment:
@smiller171 so just so I'm clear, this `extends: react-app` takes all the rules from the react config and then adds all these additional ones?